ui-components/src/commonMain/kotlin/org/jetbrains/kotlinconf/ui/components/MarkdownScreenWithTitle.kt

```diff
@@ -0,0 +1,105 @@
+package org.jetbrains.kotlinconf.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.mikepenz.markdown.compose.Markdown
+import com.mikepenz.markdown.model.DefaultMarkdownColors
+import com.mikepenz.markdown.model.DefaultMarkdownTypography
+import kotlinconfapp.ui_components.generated.resources.Res
+import kotlinconfapp.ui_components.generated.resources.arrow_left_24
+import kotlinconfapp.ui_components.generated.resources.main_header_back
+import org.jetbrains.compose.resources.stringResource
+import org.jetbrains.kotlinconf.ui.theme.KotlinConfTheme
+
+@Composable
+fun MarkdownScreenWithTitle(
+    title: String,
+    header: String,
+    subheader: String,
+    loadText: suspend () -> ByteArray,
+    onBack: () -> Unit
+) {
+    Column(Modifier.fillMaxSize()) {
+        MainHeaderTitleBar(
+            title = title,
+            startContent = {
+                TopMenuButton(
+                    icon = Res.drawable.arrow_left_24,
+                    contentDescription = stringResource(Res.string.main_header_back),
+                    onClick = onBack,
+                )
+            }
+        )
+
+        Divider(thickness = 1.dp, color = KotlinConfTheme.colors.strokePale)
+
+        Column(
+            Modifier
+                .fillMaxSize()
+                .background(color = KotlinConfTheme.colors.mainBackground)
+                .verticalScroll(rememberScrollState())
+        ) {
+
+            StyledText(header, style = KotlinConfTheme.typography.h1, modifier = Modifier.padding(12.dp))
+            if (subheader.isNotEmpty()) {
+                StyledText(
+                    subheader,
+                    style = KotlinConfTheme.typography.text2,
+                    color = KotlinConfTheme.colors.noteText,
+                    modifier = Modifier.padding(horizontal = 12.dp)
+                )
+            }
+
+            var text by remember { mutableStateOf("") }
+            LaunchedEffect(loadText) {
+                text = loadText().decodeToString()
+            }
+
+            Markdown(
```

Comment:
We need Markdown rendering for other screens as well (like the copy of the privacy policy that's a special startup screen). Could you extract this nicely set up `Markdown` call into a separate component that we can reuse on its own, by just passing in the text?

Reply:
Done, and added to that Start screen as well